bin/runner

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+user = FactoryBot.create :user, email: '123@123.nz', password: 'P@ssw0rd'
```

Comment:
You could possibly give this a more descriptive name than `runner` - maybe `user_seed`?

Reply:
Yup, will do :+1: